webpack.config.store.js

```diff
@@ -56,43 +56,43 @@ module.exports = {
             loader: "css-loader",
             options: {
               modules: false,
-              importLoaders: true
-            }
+              importLoaders: true,
+            },
           },
-          "postcss-loader"
-        ]
+          "postcss-loader",
+        ],
       },
       {
         test: /\.scss$/,
         use: [
           MiniCssExtractPlugin.loader,
           "css-loader",
           "postcss-loader",
-          "sass-loader"
-        ]
-      }
-    ]
+          "sass-loader",
+        ],
+      },
+    ],
   },
 
```

Comment:
better set prittier ignor in this file too. Many places that could get synthaxed error regarding bad commas.

Reply:
Done